shared/oae/errors/noscript.html

```diff
@@ -21,7 +21,7 @@
         </script>
 
     </head>
-    <body style="display: block;">
+    <body style="display: block; visibility: visible;">
```

Comment:
It looks like this regression was introduced as part of https://github.com/oaeproject/3akai-ux/commit/713a6898dffd85fe7a8c7aeeefeb5c90949cc114. Adding `visibility: visible` to the `body` tags should be fine. As we have switched from display to visibility, the `display: block` styles can be removed now. I'll push this as follow-up.